app/models/ext_management_system.rb

```diff
@@ -271,6 +274,7 @@ def connection_configurations=(options)
 
     delete_unused_connection_configurations(options)
   end
+  alias connection_definitions= connection_configurations=
```

Comment:
> I'd like to see some tests for this model

[providers_spec.rb L223](https://github.com/ManageIQ/manageiq/pull/9502/files#diff-c8b3e6a45e46c1b25dbf4741d3040b59R223)
[providers_spec.rb L337](https://github.com/ManageIQ/manageiq/pull/9502/files#diff-c8b3e6a45e46c1b25dbf4741d3040b59R337)

> The intention behind this is lost on me.

`connection_configurations=` is used in the system to build new `ExtManagmentSystem`s. It is used to build new EMS from UI form: [ems_common_angular.rb#L385](https://github.com/ManageIQ/manageiq/blob/master/app/controllers/mixins/ems_common_angular.rb#L385)

The problem is that `connection_configurations` does not return a struct that can be used by `connection_configurations=` , so when we do GET we can not do POST of the same struct :-(

To solve this problem, this PR introduce a new method `connection_definitions` that return the struct `connection_configurations=` expects ... so we can do GET `connection_definitions` and POST to `connection_configurations=`

To make this complete we also have `connection_definitions=` an alias to `connection_configurations=` so we can do GET and POST on the same attribute.